app/controllers/api/databases.php

```diff
@@ -2781,3 +2879,118 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('sdk.method', 'getSlowQuery')
+    ->label('sdk.description', '/docs/references/databases/get-document.md')
+    ->label('sdk.response.code', Response::STATUS_CODE_OK)
+    ->label('sdk.response.type', Response::CONTENT_TYPE_JSON)
+    ->label('sdk.response.model', Response::MODEL_DOCUMENT)
+    ->param('documentId', '', new UID(), 'Document ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->inject('mode')
+    ->inject('project')
+    ->action(function (string $documentId, Response $response, Database $dbForProject) {
+        $document = Authorization::skip(fn () => $dbForProject->getDocument('slowQueries', $documentId));
+        if ($document->isEmpty()) {
+            throw new Exception(Exception::DOCUMENT_NOT_FOUND);
+        }
+
+        $response->dynamic($document, Response::MODEL_DOCUMENT);
+    });
+
+App::delete('/v1/databases/slow-queries/:documentId')
+    ->desc('Delete Slow query Document')
+    ->desc('List  Documents')
+    ->groups(['api', 'database'])
+    ->label('docs', false)
+    ->label('sdk.hide', true)
+    ->label('scope', 'documents.write')
+    ->label('usage.metric', 'documents.{scope}.requests.delete')
+    //->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('sdk.auth', [APP_AUTH_TYPE_ADMIN])
+    ->label('sdk.namespace', 'databases')
+    ->label('event', 'slowQueries.documents.[documentId].delete')
+    ->label('audits.event', 'document.delete')
+    ->label('audits.resource', 'database/{request.databaseId}/collection/{request.collectionId}/document/{request.documentId}')
+    ->label('sdk.response.code', Response::STATUS_CODE_NOCONTENT)
+    ->label('sdk.response.model', Response::MODEL_NONE)
+    ->param('documentId', '', new UID(), 'Document ID.')
+    ->inject('response')
+    ->inject('dbForProject')
+    ->inject('events')
+    ->inject('deletes')
+    ->action(function (string $documentId, Response $response, Database $dbForProject, Event $events, Delete $deletes) {
+        $document = Authorization::skip(fn() => $dbForProject->getDocument('slowQueries', $documentId));
+        if ($document->isEmpty()) {
+            throw new Exception(Exception::DOCUMENT_NOT_FOUND);
+        }
+
+        Authorization::skip(fn() => $dbForProject->deleteDocument('slowQueries', $documentId));
+
+        $dbForProject->deleteCachedDocument('slowQueries', $documentId);
+
+        $deletes
+            ->setType(DELETE_TYPE_AUDIT)
+            ->setDocument($document)
+        ;
+
+        // todo: check slow Queries events
+
+        $events
+            ->setParam('collectionId', 'slowQueries')
+            ->setParam('documentId', $document->getId())
+            ->setPayload($response->output($document, Response::MODEL_DOCUMENT))
+        ;
```

Comment:
We can remove this, we won't need to delete audit documents or trigger events for slow queries

Reply:
Done!

Reply:
Might need to push?

Reply:
pushed..